main/glossary/README.md

```diff
@@ -67,151 +104,379 @@ synchronously compared for structural equivalence.
+in another vat, machine, or blockchain. It takes a local representative (a proxy) for a remote object as an argument and 
+sends messages to it using normal message-sending syntax. The local proxy forwards all messages to the remote object to deal with. 
+All `E()` calls return a promise for the eventual returned value. For more detail, see 
+the [`E()` section in the Distributed JavaScript page](/distributed-programming.md#communicating-with-remote-objects-using-e).
 
 ## ERTP
-*Electronic Rights Transfer Protocol* is a uniform way of transferring tokens and other digital assets, both [fungible](#fungible) and [non-fungible](#non-fungible), in JavaScript. All kinds of digital assets can easily be created and they can be all be transferred in exactly the same ways, with exactly the same security properties.
+
+*Electronic Rights Transfer Protocol* is a uniform way of transferring tokens and other digital assets, 
+both [fungible](#fungible) and [non-fungible](#non-fungible), in JavaScript. All kinds of digital assets
+can easily be created and they can be all be transferred in exactly the same ways, with exactly the same security properties.
 
 It uses [object capabilities](#object-capabilities) to enforce access control. Instead of having 
 to prove ownership of a corresponding private key, if your program has a 
 reference to an object, it can call methods on that object. If it doesn't 
 have a reference, it can't. For more on object capabilities, see [this post](http://habitatchronicles.com/2017/05/what-are-capabilities/).
 
-Key ERTP concepts include [Issuers](#issuer), [mints](#mint), 
+Key ERTP concepts include [Issuers](#issuer), [Mints](#mint), 
 [Purses](#purse), [Payments](#payment), [Brands](#brand), and [Amounts](#amounts). Also 
 see the [ERTP Introduction](/getting-started/ertp-introduction.md),
 [ERTP Guide](/ertp/guide/), and [ERTP API](/ertp/api/).
 
+## Escrow
+
+To give assets for a possible transaction to an impartial third party, who keeps them until specified conditions are satisfied. 
+For example, Alice wants to sell Bob a ticket for $100. Alice escrows the ticket, and Bob escrows the $100, with Zoe. Zoe
+does not give Alice the $100 or Bob the ticket until it has both items. Since neither Alice nor Bob ever holds both items at
+once, they don't have to trust each other to do the transaction. Zoe automatically escrows payments for transaction offers.
+
+## Eventual Send
+
+See [`E()`](#e) above.
+
+## Exit Rule
+
+Part of an [offer](#offer) specifying how the offer can be cancelled/exited. There are three values:
+- `onDemand: null`: (Default) The offering party can cancel on demand.
+- `waived: null`: The offering party can't cancel and relies entirely on the smart contract to promptly finish their offer.
+- `afterDeadline`: The offer is automatically cancelled after a deadline, as determined by its timer and deadline properties.
```

Comment:
My inclination is not to, both because those are terms whose technical definition here matches up with a standard definition, and because that's going a bit deeper than I think is needed in the glossary. Definitely in the detailed documentation, but I don't think it's worth it here.  (I was borderline on the specific rules, but decided they are specific enough terms and used as such to warrant inclusion).